examples/confidential_computing/simple/main.tf

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Since in this PR you are implementing a new variable for `instance_template` module, won't the example be better placed in `../../../examples/instance_template/confidential_computing` ?